shared/pkg/families/config.go

```diff
@@ -16,15 +16,35 @@
 package families
 
 import (
+	"time"
+
+	"github.com/anchore/syft/syft/source"
+	kubeclarityConfig "github.com/openclarity/kubeclarity/shared/pkg/config"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/openclarity/vmclarity/api/models"
 	"github.com/openclarity/vmclarity/shared/pkg/families/exploits"
+	exploitsCommon "github.com/openclarity/vmclarity/shared/pkg/families/exploits/common"
+	exploitdbConfig "github.com/openclarity/vmclarity/shared/pkg/families/exploits/exploitdb/config"
 	"github.com/openclarity/vmclarity/shared/pkg/families/malware"
+	malwareconfig "github.com/openclarity/vmclarity/shared/pkg/families/malware/clam/config"
+	malwarecommon "github.com/openclarity/vmclarity/shared/pkg/families/malware/common"
 	misconfigurationTypes "github.com/openclarity/vmclarity/shared/pkg/families/misconfiguration/types"
 	"github.com/openclarity/vmclarity/shared/pkg/families/rootkits"
+	chkrootkitConfig "github.com/openclarity/vmclarity/shared/pkg/families/rootkits/chkrootkit/config"
+	rootkitsCommon "github.com/openclarity/vmclarity/shared/pkg/families/rootkits/common"
 	"github.com/openclarity/vmclarity/shared/pkg/families/sbom"
 	"github.com/openclarity/vmclarity/shared/pkg/families/secrets"
+	"github.com/openclarity/vmclarity/shared/pkg/families/secrets/common"
+	gitleaksconfig "github.com/openclarity/vmclarity/shared/pkg/families/secrets/gitleaks/config"
 	"github.com/openclarity/vmclarity/shared/pkg/families/vulnerabilities"
 )
 
+const (
```

Comment:
Are these default values? If so, please add `Default` prefix to their names. Otherwise these do not belong here in my opinion.